users.json

```
@@ -44,4 +44,9 @@
     "role": "admin",
     "publicKey": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJtnZ3DbWbrDAMJ9lmy/8Ny6DHNNTk31ec9X3wEXkj/9"
   }
```

```suggestion
  },
```

We should add a workflow for PRs too that runs something like `tofu validate`, you are missing a comma here.

---

Fixed